DFC.ServiceTaxonomy.GraphVisualiser/Views/Visualise/Viewer.cshtml

```diff
@@ -0,0 +1,558 @@
+                        <p class="propDetails">Properties: <span id="nodeProperties"></span></p>
+                    </div>
+                    <div id="propertySelectionInformation" class="hidden">
+                        <p class="propDetails">Name: <span id="propname"></span></p>
+                        <p class="propDetails">Type: <span id="typeProp"></span></p>
+                        <p id="inverse" class="propDetails">Inverse: <span></span></p>
+                        <p class="propDetails">Domain: <span id="domain"></span></p>
+                        <p class="propDetails">Range: <span id="range"></span></p>
+                        <p class="propDetails">Subprop.: <span id="subproperties"></span></p>
+                        <p class="propDetails">Superprop.: <span id="superproperties"></span></p>
+                        <p class="propDetails">Equiv.: <span id="propEquivUri"></span></p>
+                        <p id="infoCardinality" class="propDetails">Cardinality: <span></span></p>
+                        <p id="minCardinality" class="propDetails">Min. cardinality: <span></span></p>
+                        <p id="maxCardinality" class="propDetails">Max. cardinality: <span></span></p>
+                        <p class="propDetails">Charac.: <span id="propAttributes"></span></p>
+                        <p class="propDetails">Description: <span id="propDescription"></span></p>
+                        <p class="propDetails">Comment: <span id="propComment"></span></p>
+                    </div>
+                    <div id="noSelectionInformation">
+                        <p><span>Select an element in the visualization.</span></p>
+                    </div>
+                </div>
+            </section>
+        </aside>
+        <div id="blockGraphInteractions" class="hidden"></div>
+        <div id="WarningErrorMessagesContainer" class="">
+            <div id="WarningErrorMessages" class="">
+            </div>
+        </div>
+
+        <div id="DirectInputContent" class="hidden">
+            <div id="direct-text-input-container">
+                <textarea rows="4" cols="25" title="Direct Text input as JSON(experimental)" class="directTextInputStyle" id="directInputTextArea"></textarea>
+                <div id="di_controls">
+                    <ul>
+                        <li>
+                            <button id="directUploadBtn">Upload</button>
+                        </li>
+                        <li>
+                            <button id="close_directUploadBtn">Close</button>
+                        </li>
+                        <li id="Error_onLoad" class="hidden">Some text if ERROR</li>
+                    </ul>
+                </div>
+            </div>
+
+
+        </div>
+    </main>
+    <aside id="class-properties-popup" class="avgrund-popup">
+        <h2 id="class-properties-popup-label">Class label</h2>
+        <p id="class-properties-popup-properties">
+            properties here
```

Comment:
Looks like we didn't end up using avgrund, so we can remove this and its js / css files.

Reply:
ok

Reply:
removed